Dockerfile-confluenthub

```diff
@@ -0,0 +1,22 @@
+#
+# Copyright 2020 Confluent Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+FROM confluentinc/cp-server-connect:5.5.x-latest
```

Comment:
Should this image build on `cp-server-connect:5.4.1` instead of `cp-server-connect:5.5.x-latest`?

Reply:
@mikebin going back to the "QA" aspect, I'd like to keep as many components as possible on 5.5.x (or whatever the current branch is).  In this demo, 5.4.1 Replicator works with 5.5.x Connect worker -- is that on accident or is that supposed to work?

Reply:
I agree with using as much 5.5.x as possible, but was the other `Dockerfile-local` intended for that case where we test with _all_ 5.5.x components? I figured `Dockerfile-confluenthub` was intended for running the release version components of whatever is the latest version. Not sure if verifying that 5.4.1 Replicator runs on 5.5.x Connect is critical (although it doesn't hurt).

Bottom line, I don't feel strongly about this, so I'm fine with whatever you think is most appropriate.

Reply:
@mikebin 

* Dockerfile-confluenthub: test with GA version of Replicator/ES (default)
* Dockerfile-local: test with all 5.5.x components, including Replicator/ES